src/libraries/sendtohelixhelp.proj

```diff
@@ -233,14 +232,35 @@
     </ItemGroup>
   </Target>
 
+  <PropertyGroup>
+    <_ShouldSetXUnitLogCheckerEnvVar>0</_ShouldSetXUnitLogCheckerEnvVar>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(IsXUnitLogCheckerSupported)' == 'true'">
```

Comment:
Why don't you move the `BuildTargetFramework` condition up into this PropertyGroup so that all these env vars are only set when necessary?

Reply:
Hmm good point. There's no .NET Framework in unix, so that should work.